src/ucp/core/ucp_worker.c

```diff
@@ -887,6 +887,16 @@ static int ucp_worker_iface_support_keepalive(ucp_worker_iface_t *wiface)
                               UCT_IFACE_FLAG_EP_KEEPALIVE);
 }
 
+/* Compare target iface atomic capabilities to candidate's, and decide if candidate is better */
+static int ucp_worker_iface_is_better_atomic_caps(uct_iface_attr_t *cand_attr,
```

Comment:
Why pass ptr to attr and no to the caps?

Reply:
`cap` is an implicit struct inside `uct_iface_attr_t` (it has no type name).
is it possible to reference this kind of struct as a pointer?

Reply:
Right. Then let's leave it with `attr`.